packages/db/src/runtime/seed-local.ts

```diff
@@ -3,7 +3,7 @@ import { type SQL, sql } from 'drizzle-orm';
 import type { LibSQLDatabase } from 'drizzle-orm/libsql';
 import { SQLiteAsyncDialect } from 'drizzle-orm/sqlite-core';
 import { type DBTables } from '../core/types.js';
-import { AstroDbError } from '../utils.js';
```

Comment:
Wow, I'm surprised this would cause an issue. `utils.js` is at the project root, _not_ in the `core` directory